docs/installation/migrate.rst

```diff
@@ -16,77 +16,130 @@ You can set up continuous migration of |sr| to maintain a hybrid deployment ("ex
+    other services are down, as they can conflict with schema migration.
+    
+    ::
+    
+         control-center is [DOWN]
+         ksql-server is [DOWN]
+         connect is [DOWN]
+         kafka-rest is [DOWN]
+         schema-registry is [UP]
+         kafka is [UP]
+         zookeeper is [UP]
 
-        curl -X PUT -H "Content-Type: application/json" -H "Authorization: Basic dGVuYW50Mi1rZXk6bm9oYXNo" "http://destregistry:8081/mode" --data '{"mode": "IMPORT"}'
+#.  Set the destination |sr| to IMPORT mode.  For example: 
 
-#.  Configure Replicator with Schema Registry information.
+    .. code:: bash
+    
+        curl -u <schema-registry-api-key>:<schema-registry-api-secret> -X PUT -H "Content-Type: application/json" "http://destregistry:8081/mode" --data '{"mode": "IMPORT"}'
+
+#.  Configure a |crep| worker to specify the addresses of broker(s) in the destination cluster, as described in :ref:`config-and-run-replicator`.
+
+    The worker configuration file is in `<path-to-confluent>/etc/kafka/connect-standalone.properties`.
+
+    :: 
 
+    # Connect Standalone Worker configuration
+    bootstrap.servers=localhost:9092
+                
+#.  Configure :ref:`replicator` <replicator-quickstart>` with |sr| and destination cluster information.
+
+    For example, set the following in `etc/kafka-connect-replicator/quickstart-replicator.properties`:
 
     :: 
+
+        # destination cluster connection info
+        dest.kafka.ssl.endpoint.identification.algorithm=https
+        dest.kafka.sasl.mechanism=PLAIN
+        dest.kafka.request.timeout.ms=20000
+        dest.kafka.bootstrap.servers=<path-to-cloud-server>.cloud:9092
+        retry.backoff.ms=500
+        dest.kafka.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="2OSKHFEI22T7EJ7W" password="+UQUxhdL+R4hkQhzoUrtZg3FXwx/uLchTtXQRZ2NTGXd2zeQNeiBbaNe231tF7F+";
+        dest.kafka.security.protocol=SASL_SSL    
     
+        # Schema Registry migration topics to replicate from source to destination
+        topic.whitelist=_schemas
         schema.topic=_schemas
-        schema.registry.url=http://somehost:8081
+        
+        # Connection settings for destination Confluent Cloud Schema Registry
+        schema.registry.url=https://<path-to-cloud-schema-registry>.cloud
         schema.registry.client.basic.auth.credentials.source=USER_INFO
-        schema.registry.client.basic.auth.user.info=<user>:<password>
-  
-    
-#.  Start Replicator so that it can perform the one-time schema migration. 
+        schema.registry.client.basic.auth.user.info=<schema-registry-api-key>:<schema-registry-api-secret>
 
-    .. code:: bash
-    
-        TBD example "start replicator" command
+    .. tip:: In `quickstart-replicator.properties`, the replication factor is set to `1` for demo purposes. For this schema migration tutorial, and in production, change this to at least 3: `confluent.topic.replication.factor=3`
```

Comment:
@rayokota : why is `confluent.topic.replication.factor` relevant?  Is the schemas topic being copied as well?  Why?